assets/app/lib/hex.rb

```diff
@@ -21,6 +27,32 @@ module Hex
       "M #{X_R},#{Y_M} L #{X_M_R}, #{Y_B}",
     ].freeze
 
+    INTERSECT = Y_T + ((5 * STRIPE_WIDTH - X_M_R).to_f * Y_T.to_f / (X_M_R - X_R).to_f).to_i
+    # 3 lists of points
+    STRIPE_POINTS = [
+      [[-1 * STRIPE_WIDTH, Y_T], [STRIPE_WIDTH, Y_T], [STRIPE_WIDTH, -1 * Y_T], [-1 * STRIPE_WIDTH, -1 * Y_T]],
+      [
+        [3 * STRIPE_WIDTH, Y_T],
+        [X_M_R, Y_T],
+        [5 * STRIPE_WIDTH, INTERSECT],
+        [5 * STRIPE_WIDTH, -1 * INTERSECT],
+        [X_M_R, -1 * Y_T],
+        [3 * STRIPE_WIDTH, -1 * Y_T],
+],
```

Comment:
formatting

Reply:
I've indented this and the other affected line